### PR TITLE
Remove java runtime from overview

### PR DESCRIPTION
**Pull request by akaroml**
Signed-off-by: Rome Li <rome.li@microsoft.com>

### CODE REVIEW

File: src/java-runtime/index.ts [hunk 1]
```diff
@@ -175,14 +181,18 @@ export async function findJavaRuntimeEntries(): Promise<JavaRuntimeEntry[]> {
   return Promise.all(_.map(entries, async entry => {
     if (!await checkJdkInstallation(entry.path)) {
       entry.isValid = false;
-      entry.hint = "This path does not point to a JDK.";
+      entry.hint = "This path is not pointing to a JDK.";
+      const pathObject = path.parse(entry.path||"");
+      if (pathObject.name && pathObject.name === "bin") {
```
Comment (jdneo) [inline src/java-runtime/index.ts]: nit: `pathObject.name === "bin"` is enough. Leave the choice to you

File: src/java-runtime/index.ts [hunk 2]
```diff
@@ -175,14 +181,18 @@ export async function findJavaRuntimeEntries(): Promise<JavaRuntimeEntry[]> {
   return Promise.all(_.map(entries, async entry => {
     if (!await checkJdkInstallation(entry.path)) {
       entry.isValid = false;
-      entry.hint = "This path does not point to a JDK.";
+      entry.hint = "This path is not pointing to a JDK.";
+      const pathObject = path.parse(entry.path||"");
+      if (pathObject.name && pathObject.name.toLowerCase() === "bin") {
```
Comment (akaroml) [inline src/java-runtime/index.ts]: Case insensitive comparison is what I really wanted in the first place...

File: src/java-runtime/index.ts (hunk 1 above)
Comment (akaroml) [reply src/java-runtime/index.ts]: Addressed. Please check the new commit.

File: src/java-runtime/index.ts (hunk 2 above)
Comment (Eskibear) [reply src/java-runtime/index.ts]: any concern of using `path.basename(string)` instead of `path.parse(string)`?

Comment (akaroml) [reply src/java-runtime/index.ts]: works too. But let's move on...

**Pull request merged**